app/scripts/services/Room.js

```diff
@@ -3,8 +3,17 @@
 		var ref = firebase.database().ref().child("rooms");
 		var rooms = $firebaseArray(ref);
 
+		var addRoom = function(roomName) {
+			if (/\S/.test(roomName)) {
+				rooms.$add({room: roomName});
+			} else {
+				return window.alert("Use your words, please!");	
```

Comment:
This also belongs on the controller. An `alert` is ok for dev stage but ultimately this would be better as inline feedback.